### DISCUSSION TITLE
Help On Usage

### DISCUSSION THREAD

**Discussion by slayerprogrammer** [quartznet/quartznet#1205] [Q&A] [answered]
Hello,
I am using Quartz.net in my .Net Core project. In my project, vehicle sales are made on a tender basis. The tender end time of each vehicle is at different times. When the tender period is over, I want to update the related tender (id) status. But I don't know how I can do this with Quartz.net. Can you help me ?

**Comment by lahma** [comment, marked as answer]
You could probably store a single-shot trigger per tender or create recurring trigger that scans database for actions. For first option you need persistent store and with second option you could probably get away without one at least at the beginning if you handle state in your own application database.
